package.json

```diff
@@ -3,14 +3,21 @@
     "displayName": "Kubernetes",
     "description": "Develop, deploy and debug Kubernetes applications",
     "version": "0.1.0",
-    "publisher": "Microsoft",
+    "preview": true,
+    "publisher": "ms-kubernetes-tools",
     "engines": {
         "vscode": "^1.13.0"
     },
     "license": "MIT",
     "categories": [
+        "Azure",
         "Other"
     ],
+    "keywords": [
+        "kubernetes",
+        "k8s",
+        "azure"
```

Comment:
`helm` would belong here, then.

Reply:
if draft commands are here, then you want draft in the keywords as well as helm.